# jcf-parse.o needs -save-temps on m68k-linux, else it segfaults on bootstrap

--- src/gcc/java/Make-lang.in~	Wed Apr 10 15:08:58 2002
+++ src/gcc/java/Make-lang.in	Sat May 11 10:51:22 2002
@@ -304,6 +304,11 @@
   $(SYSTEM_H)
 java/zextract.o: java/zextract.c $(CONFIG_H) $(SYSTEM_H) java/zipfile.h
 
+# jcf-parse.o needs -save-temps on m68k-linux, else it segfaults
+java/jcf-parse.o: java/jcf-parse.c $(CONFIG_H) $(SYSTEM_H) $(JAVA_TREE_H)
+	$(CC) -c $(ALL_CFLAGS) $(ALL_CPPFLAGS) $(INCLUDES) \
+		$(srcdir)/java/jcf-parse.c $(OUTPUT_OPTION)
+
 # jcf-io.o needs $(ZLIBINC) added to cflags.
 java/jcf-io.o: java/jcf-io.c $(CONFIG_H) $(SYSTEM_H) $(JAVA_TREE_H)
 	$(CC) -c $(ALL_CFLAGS) $(ALL_CPPFLAGS) $(INCLUDES) $(ZLIBINC) \
